crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -887,10 +882,20 @@ enum IbcSudoChangeErrorKind {
 ///
 /// It also contains a `return_address` field which may or may not be the same as the signer
 /// of the packet. The funds will be returned to the `return_address` in the case of a timeout.
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Ics20Withdrawal {
+#[derive(Debug, Clone, PartialEq)]
+pub enum Ics20Withdrawal {
+    NoBridgeAddress(Box<Ics20WithdrawalNoBridgeAddress>),
```

Comment:
would prefer naming like `FromSequencer` to match `FromRollup`

Reply:
Changed in 7a5329c :)